packages/ui/dialogs/my-account/index.tsx

```diff
@@ -372,6 +373,30 @@ class MyAccount extends Component<Props, State> {
     )
   }
 
+  renderCryptoBalancesSubpanel() {
+    const { ethBalance, bcptBalance } = this.props.state
+
+    const cryptoSubpanels = ERC20_Tokens.map( (token) => {
+      return (
+        <View style={style.spaceHorizontalL}>
```

Comment:
the top-level view needs a `key` property, I suggest using the index of the array. So `(token) =>` would become `(token, index) =>`

Reply:
Also, line 200 of `my-account` should be changed to `const panelHeights = [150, 250, 60, 60, 60, 60, 140, 60, 190, 180, 60, 120]` (it should be of length 12 now that you've removed one of the options)

Reply:
good find